src/pages/editor.js

```diff
@@ -18,7 +16,7 @@ const getParam = function (val) {
   let tmp = [];
 
   window.location.search
-    .substr(1)
+    .substring(1)
```

Comment:
I was getting the message at `.substr` was depreciated so I updated it to `.substring(1)` which should work the same since we're passing in one arg.